translate/cloud-client/src/main/java/com/example/translate/HybridGlossaries.java

```diff
@@ -0,0 +1,301 @@
+                      .setGlossary(glossaryName)
+                      .build();
+
+      TranslateTextGlossaryConfig translateTextGlossaryConfig =
+              TranslateTextGlossaryConfig.newBuilder()
+                      .setGlossary(glossaryId.toString())
+                      .build();
+
+      TranslateTextRequest translateTextRequest =
+              TranslateTextRequest.newBuilder()
+                      .setParent(locationName.toString())
+                      .setMimeType("text/plain")
+                      .setSourceLanguageCode(sourceLanguageCode)
+                      .setTargetLanguageCode(targetLanguageCode)
+                      .addContents(text)
+                      .setGlossaryConfig(translateTextGlossaryConfig)
+                      .build();
+
+      // Call the API
+      TranslateTextResponse response = translationServiceClient.translateText(translateTextRequest);
+      return response.getGlossaryTranslationsList().get(0).getTranslatedText();
+    } catch (Exception e) {
+      throw new RuntimeException("Couldn't create client.", e);
+    }
+  }
+  // [END translate_hybrid_translate]
+
+  // [START translate_hybrid_tts]
+  /**
+   * Generates synthetic audio from plaintext.
+   *
+   * Given a string of plaintext, this function converts the contents of the input text file
+   * into a String of tagged SSML text. This function formats the SSML String so that,
+   * when synthesized, the synthetic audio will pause for two seconds between each line
+   * of the text file. This function also handles special text characters which might
+   * interfere with SSML commands.
+   * Given a string of SSML text and an output file name, this function
+   * calls the Text-to-Speech API. The API returns a synthetic audio
+   * version of the text, formatted according to the SSML commands. This
+   * function saves the synthetic audio to the designated output file.   
+   *
+   * @param text plaintext input
+   * @param outFile String name of file under which to save audio output
+   *
+   */
+  static void textToAudio(String text, String outFile) {
+    // Replace special characters with HTML Ampersand Character Codes
+    // These codes prevent the API from confusing text with SSML tags
+    // For example, '<' --> '&lt;' and '&' --> '&amp;'
+    String escapedLines = HtmlEscapers.htmlEscaper().escape(text);
+
+    // Convert plaintext to SSML
+    // Tag SSML so that there is a 2 second pause between each address
+    String expandedNewline = escapedLines.replaceAll("\\n","\n<break time='2s'/>");
+    String ssmlText = "<speak>" + expandedNewline + "</speak>";
+
+    // Instantiates a client
+    try (TextToSpeechClient textToSpeechClient = TextToSpeechClient.create()) {
+      // Set the ssml text input to synthesize
+      SynthesisInput input = SynthesisInput.newBuilder().setSsml(ssmlText).build();
+
+      // Build the voice request, select the language code ("en-US") and
+      // the ssml voice gender ("male")
```

Comment:
Some of these comments are broken in weird places. The comment should go to the line limit.